nl_server/config_reader.py

```diff
@@ -44,17 +44,9 @@ def _path_from_current_file(rel_path: str) -> str:
   return os.path.join(os.path.dirname(__file__), rel_path)
 
 
-# catalog paths
-
-# catalog from checked in file
-_CATALOG_CODE_PATH = _path_from_current_file('../deploy/nl/catalog.yaml')
-# catalog from mounted file, this is for GKE deployments
-_CATALOG_MOUNT_PATH = '/datacommons/nl/catalog.yaml'
-# catalog from user provided file, this is for custom DCs
-# TODO: use absolute path set by user.
-_CATALOG_USER_PATH_SUFFIX = 'datacommons/nl/custom_catalog.yaml'
-
-# env paths
+# Default catalog paths to load from
```

Comment:
what are these default paths are for different envs?

Reply:
updated in follow up PR